Why operation canceled after 1h

I am building big job and "The operation was canceled."
and in gitea it is still "running."

> MESSAGE: {
    "Count": 2,
    "Value": [
        "_CppCoreCompile:",
        "  C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\bin\\bcc32.exe -D_DEBUG;XE -D_RTLDLL;USEPACKAGES -n..\\_Debug\\Win32 -I..\\Ciselniky\\DMP;..\\KlinModuly\\RTG;..\\Ciselniky\\UzvUnicisel;..\\Ciselniky\\Hospit;..\\Ciselniky\\Images;..\\ObecDotaz\\Konfig;..\\Ciselniky\\Oracle;..\\Ciselniky\\Ostatni;..\\Ciselniky\\Uctovani;..\\Ciselniky\\Fakturace;..\\Vyvolani;..\\Reporty;..\\KlinModuly;..\\PojistPoriz;..\\Hospit;..\\DRGExpert;..\\VykZum;..\\Sklady;..\\PdfTexty;..\\Rozvrh;..\\Rights;..\\Zadanky;..\\Ciselniky\\Leky;..\\Ciselniky\\Pracoviste;..\\LabView;..\\Ciselniky;..\\Ciselniky\\Uzivatele;..\\Ciselniky\\TypUdal;..\\Ciselniky\\TypProc;..\\Ciselniky\\Pristroje;..\\Ciselniky\\Import;..\\Ciselniky\\Prava;..\\Komunikace;..\\Common;..\\Ciselniky\\ExtZar;..\\Ciselniky\\EditModuly;..\\Generic;..\\Pacienti;..\\Diagnozy;..\\Main;..\\AmbPece;..\\Debug;..\\ZdrUct;..\\Certifikaty;..\\OrdinLeky;..\\ObecDotaz;..\\ZadankaModuly\\Laborator;\"C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\include\";\"C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\include\\windows\\crtl\";\"C:\\Program Files (x86)\\E..."
    ],
    "StepID": "eaca35e6-9ec5-4dc1-af09-f61c8b5c57ae",
    "StartLine": 631
}
MESSAGE: {
    "Count": 2,
    "Value": [
        "_CppCoreCompile:",
        "  C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\bin\\bcc32.exe -D_DEBUG;XE -D_RTLDLL;USEPACKAGES -n..\\_Debug\\Win32 -I..\\Ciselniky\\DMP;..\\KlinModuly\\RTG;..\\Ciselniky\\UzvUnicisel;..\\Ciselniky\\Hospit;..\\Ciselniky\\Images;..\\ObecDotaz\\Konfig;..\\Ciselniky\\Oracle;..\\Ciselniky\\Ostatni;..\\Ciselniky\\Uctovani;..\\Ciselniky\\Fakturace;..\\Vyvolani;..\\Reporty;..\\KlinModuly;..\\PojistPoriz;..\\Hospit;..\\DRGExpert;..\\VykZum;..\\Sklady;..\\PdfTexty;..\\Rozvrh;..\\Rights;..\\Zadanky;..\\Ciselniky\\Leky;..\\Ciselniky\\Pracoviste;..\\LabView;..\\Ciselniky;..\\Ciselniky\\Uzivatele;..\\Ciselniky\\TypUdal;..\\Ciselniky\\TypProc;..\\Ciselniky\\Pristroje;..\\Ciselniky\\Import;..\\Ciselniky\\Prava;..\\Komunikace;..\\Common;..\\Ciselniky\\ExtZar;..\\Ciselniky\\EditModuly;..\\Generic;..\\Pacienti;..\\Diagnozy;..\\Main;..\\AmbPece;..\\Debug;..\\ZdrUct;..\\Certifikaty;..\\OrdinLeky;..\\ObecDotaz;..\\ZadankaModuly\\Laborator;\"C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\include\";\"C:\\Program Files (x86)\\Embarcadero\\RAD Studio\\8.0\\include\\windows\\crtl\";\"C:\\Program Files (x86)\\E..."
    ],
    "StepID": "eaca35e6-9ec5-4dc1-af09-f61c8b5c57ae",
    "StartLine": 633
}
MESSAGE: {
    "Count": 2,
    "Value": [
        "context canceled",
        "##[error]The operation was canceled."
    ],
    "StepID": "eaca35e6-9ec5-4dc1-af09-f61c8b5c57ae",
    "StartLine": 635
}
MESSAGE: {
    "Count": 2,
    "Value": [
        "Post job cleanup.",
        "##[command]C:\\Program Files\\Docker\\Docker\\resources\\bin\\docker.EXE exec  c36453e7cba88c114741f1d2b6d11b50da7509cb3df689efec59f42aaf0eb859 sh -c \"cat /etc/*release | grep ^ID\""
    ],
    "StepID": "1bbbf3fe-580c-4319-a92e-5ec1052b2f47",
    "StartLine": 1
}

--- a/poller/poller.go
+++ b/poller/poller.go
@@ -193,8 +193,5 @@ func (p *Poller) dispatchTask(ctx context.Context, task *runnerv1.Task) error {
 		}
 	}()
 
-	runCtx, cancel := context.WithTimeout(ctx, time.Hour)
-	defer cancel()
-
-	return p.Dispatch(runCtx, task)
+	return p.Dispatch(ctx, task)
 }

